comp/core/flare/flare.go

```diff
@@ -140,14 +141,19 @@ func (f *flare) createAndReturnFlarePath(w http.ResponseWriter, r *http.Request)
 		}
 	}
 
+	queryProviderTimeout := r.URL.Query().Get("provider_timeout")
+	providerTimeoutSeconds, err := strconv.Atoi(queryProviderTimeout)
+	if err != nil {
```

Comment:
Did you consider making the difference between `queryProviderTimeout` to be empty (which valid value) and `queryProviderTimeout`  cannot be converted to an int (which is an error)?

Reply:
I'll add a warning if a value is given but is invalid